gallery/usage/microsoft.insights-components.json

```diff
@@ -55,6 +55,12 @@
 					"id": "Workbooks/Usage/User Retention Analysis",
 					"author": "Microsoft",
 					"name": "User Retention Analysis"
+				},
+				{
+					"id": "Workbooks/Usage/Heart",
+					"author": "Microsoft",
+					"name": "Heart - Product Usage Measurement",
```

Comment:
There was feedback provided offline to add some clarity in the gallery title for users to know this is specific to the click analytics plugin (ex. `Product Usage Measurement for Click Analytics`). Do we plan to change it or was it decided this is the way it should be?